src/data/repositories/utils/MetadataHelper.ts

```diff
@@ -20,3 +21,18 @@ export function getProgramTEAsMetadata(api: D2Api, programId: string) {
         })
     );
 }
+
+export function getUserGroupsByCode(api: D2Api, code: string): FutureData<Ref[]> {
```

Comment:
only one usergroup can be returned as it is filtered by code , so rename to getUserGroupByCode and return type should be Ref